code/modules/maps/lateload/station_specific/_tether_lateload.dm

```diff
@@ -0,0 +1,70 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// Code Shenanigans for Tether lateload maps
+/datum/map_template/tether_lateload
+	allow_duplicates = FALSE
+	var/associated_map_datum
```

Comment:
So many individual defines of `var/associated_map_datum` tells me we need to put it on `map_template`

Reply:
Probably would be more efficient to just do /datum/map_template/lateload/ and have it on that. Putting it on map_template as a base might jack some stuff up as it might try assigning a z level to every submap for example

Reply:
Sounds good to me

Reply:
Done